src/main/java/com/example/cats/domain/Product.java

```diff
@@ -0,0 +1,19 @@
+package com.example.cats.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
```

Comment:
you shouldn't use the both builder and constructor + you can stick to one, as in the end it's just 2 ways of creating your objects